bluebird.d.ts

```diff
@@ -151,8 +151,8 @@ declare class Bluebird<R> implements Bluebird.Thenable<R>, Bluebird.Inspection<R
    * });
    * </code>
    */
-  // TODO find way to fix get()
-  // get<U>(propertyName: string): Bluebird<U>;
+  // TODO: explain original comment from DT: TODO find way to fix get()
```

Comment:
I think they'll be in 2.0 or something like that. agreed we shouldn't wait for them. I was just explaining the TODO comment.